docs/release/release.md

```diff
@@ -0,0 +1,430 @@
+# K3S Release Process
+## Setup Environment
```

Comment:
We might want to split this section out to its own file and reference that file here.

Reply:
done